dotcom-rendering/src/components/CartoonComponent.tsx

```diff
@@ -25,17 +29,36 @@ export const CartoonComponent = ({ format, element, switches }: Props) => {
 	const render = (image: Image) => {
 		return (
 			<>
-				<Picture
-					master={image.url}
-					role={element.role}
-					format={format}
-					alt={`${element.alt ? `${element.alt}, ` : ''}panel ${
-						image.index + 1
-					}`}
-					height={parseInt(image.fields.height, 10)}
-					width={parseInt(image.fields.width, 10)}
-					key={image.index}
-				/>
+				{renderingTarget === 'Apps' ? (
+					<Island priority="critical">
```

Comment:
I really think this is a feature, and should be marked as such!

```suggestion
					<Island priority="feature" defer={{ until: 'visible' }}>
```

Reply:
I picked this up from the lightbox behaviour on regular images (without thinking!), not sure if @georgeblahblah wrote that but adding him in case there was another reason this needed to be critical?

Reply:
Can be addressed in a separate PR if it’s consistent!